app/services/preserved_object_handler_results.rb

```diff
@@ -35,7 +36,8 @@ class PreservedObjectHandlerResults
     PC_STATUS_CHANGED => "PreservedCopy status changed from %{old_status} to %{new_status}",
     UNEXPECTED_VERSION => "incoming version (%{incoming_version}) has unexpected relationship to %{addl} db version; ERROR!",
     INVALID_MOAB => "Invalid moab, validation errors: %{addl}",
-    PC_PO_VERSION_MISMATCH => "PreservedCopy online moab version %{pc_version} does not match PreservedObject current_version %{po_version}"
+    PC_PO_VERSION_MISMATCH => "PreservedCopy online moab version %{pc_version} does not match PreservedObject current_version %{po_version}",
+    ONLINE_MOAB_DOES_NOT_EXIST => "db has moab that is not found online"
```

Comment:
I might include the preserved_copy version in the catalog, and its create date and updated_at date ???

Reply:
Naomi and I talked F2F created ticket #509 to ask Ben for more information.